support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -68,6 +72,27 @@ export const productCatalogDescription = {
 		label: 'Home Delivery',
 	},
 };
+export const productCatalogDescInclOffers = {
+	...productCatalogDescExclOffers,
+	SupporterPlusWithGuardianWeekly: {
+		label: productCatalogDescExclOffers.SupporterPlusWithGuardianWeekly.label,
+		benefitsSummary: ['The rewards from All-access digital'],
+		offersSummary: [
+			{ strong: true, copy: 'including a free book as our gift to you.†' },
+		],
+		benefits:
+			productCatalogDescExclOffers.SupporterPlusWithGuardianWeekly.benefits,
+	},
+	SupporterPlus: {
+		label: productCatalogDescExclOffers.SupporterPlus.label,
+		benefits: productCatalogDescExclOffers.SupporterPlus.benefits,
+		offers: [
+			{
+				copy: <OfferBook></OfferBook>,
+			},
+		],
+	},
+};
```

Comment:
Not a biggy, but I wonder if we could keep this transformation at the point of need AKA [in the `threeTierLanding`](https://github.com/guardian/support-frontend/pull/5896/files#diff-9e9d19a3470db4ff48cd4c73865be05d061001afc0751360451d6d509f390e9bR346-R351)?

I'm just trying to think how to avoid putting test data into this abstraction.

Reply:
Moved this into `threeTierLanding` just above the `getCardData` where its used. Done.